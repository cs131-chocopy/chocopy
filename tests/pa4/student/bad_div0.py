i:int=10
a:int=114514
while True:
        print(a%i)
        i=i-1
