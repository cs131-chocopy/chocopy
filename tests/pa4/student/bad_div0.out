4
7
2
1
4
4
2
1
0
0
Division by zero
